### PR TITLE
feat: adding pronouns to aid requests

### PR DESCRIPTION
**Pull request by asbhaskar**
After this gets merged we can use the same code to add pronouns to the volunteer page

### CODE REVIEW

**Review by jadeallencook — CHANGES_REQUESTED** [review]
Overall this looks great, I'd only suggest using all lowercase for the values. 

```js
{
    he: 'he',
    she: 'she',
    they: 'they',
}

```

We can capitalize the values easily by changing the style, but I think this will be commonly be concatenated within a sentence like this.

```js
`User has invited you to an event that ${subjectProun} is coordinating.`
```

**Comment by jadeallencook** [comment]
Oh, this is for the requester and not the volunteer. 

In addition to lowercasing the pronouns, please use this the input group in the form:
https://react-bootstrap.github.io/forms/input-group/

**Comment by jadeallencook** [comment]
We also need to display this information in the request modal used by the volunteer.

File: src/forms/ViewRequestForm/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Accordion, Button, Form, Modal } from 'react-bootstrap';
+import { Accordion, Button, Form, Modal, Row } from 'react-bootstrap';
```
Comment (asbhaskar) [inline src/forms/ViewRequestForm/index.tsx:1]: I can't find the file but we're not checking of unused variables right now, we should add that to the precommit or eslint config

**Review by jadeallencook — CHANGES_REQUESTED** [review]
This looks great!

Only suggestion is using the same form group in the request.

Screenshot 2023-05-30 at 10 30 06 PM

Also, what if someone goes by she/they or he/they?

Is that the same as she/them and he/them?

Edit: I think we should just add "they" to the object pronoun.

**Comment by asbhaskar** [comment]
I tried searching for how other sites offer this feature, a single dropdown with subj/obj and an extra other option with a conditional text input seems standard. How's that sound?

**Comment by jadeallencook** [comment]
> I tried searching for how other sites offer this feature, a single dropdown with subj/obj and an extra other option with a conditional text input seems standard. How's that sound?

That'll work, either way!

**Comment by jadeallencook** [comment]
I'm going to close this out for now but let's revisit this later.

**Pull request closed without merging**